lib/miq_automation_engine/models/miq_ae_domain.rb

```diff
@@ -103,6 +103,10 @@ def squeeze_priorities
     MiqAeDomain.reset_priority_by_ordered_ids(ids)
   end
 
+  def self.any_enabled?
```

Comment:
I think the problem we were running into before was that I was checking for "system" domains but not enabled/disabled. We might be able to use `enabled_domains` but it might not be as nice as having a boolean method.